funnyvote/app/src/androidTest/java/com/heaton/funnyvote/data/VoteData/VoteDataDaoTest.java

```diff
@@ -0,0 +1,99 @@
+package com.heaton.funnyvote.data.VoteData;
+
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.google.common.collect.Lists;
+import com.heaton.funnyvote.database.DaoMaster;
+import com.heaton.funnyvote.database.DaoSession;
+import com.heaton.funnyvote.database.Option;
+import com.heaton.funnyvote.database.VoteData;
+import com.heaton.funnyvote.database.VoteDataDao;
+
+import org.greenrobot.greendao.database.Database;
+import org.greenrobot.greendao.identityscope.IdentityScopeType;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.List;
+
+import static com.heaton.funnyvote.FunnyVoteApplication.ENCRYPTED;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@RunWith(AndroidJUnit4.class)
```

Comment:
this is a android instrument unit test. that's why the jacoco coverage is not worked